es-app/src/Gamelist.cpp

```diff
@@ -259,6 +263,9 @@ void updateGamelist(SystemData* system)
 			if (!doc.save_file(xmlWritePath.c_str())) {
 				LOG(LogError) << "Error saving gamelist.xml to \"" << xmlWritePath << "\" (for system " << system->getName() << ")!";
 			}
+
+			const auto endTs = std::chrono::system_clock::now();
+			LOG(LogInfo) << "Saved gamelist.xml for system \"" << system->getName() << "\" in " << std::chrono::duration_cast<std::chrono::milliseconds>(endTs - startTs).count() << " ms";
```

Comment:
Is this needed? This feels like something more from the performance debug stage, unsure if there's a reason to keep this?

Reply:
You are correct, it is not needed. But since saving on e.g. a RPi takes some time I thought it would be nice having a chance being able to see an explanation for the "hang" in the log.
Ofc I am fine with removing it if you guys don't want it.

Reply:
I don't have a strong opinion either way. Maybe @tomaz82 or @jrassa will. If no further comments, just keep it.

Reply:
I'd say remove this